pkg/kn/commands/source/list_test.go

```diff
@@ -71,6 +79,20 @@ func TestSourceListTypesNoHeaders(t *testing.T) {
 	assert.Check(t, util.ContainsAll(output[0], "PingSource"))
 }
 
+func TestListBuiltInSources(t *testing.T) {
```

Comment:
I will suggest to use `TestListBuiltInSourceTypes` other than `TestListBuiltInSources` because the method is to test listing of builtin source types, other than sources.

Reply:
updated